### DISCUSSION TITLE
Show BOM parameters

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7742] [General]
Hi,
   I have built a build with a BOM but something a bit annoying is that I'm not able to show the parameters of the BOMs in one table and there is no option there, is there a way to workardound this? The BOM seems to be stock centric but not very parameter and comparison centric.

**Comment by SchrodingersGat** [comment]
@jesseyu1984918 potentially this could be added in, either to the existing BOM table or as a separate table. But currently, this option does not exist.

We would also only consider adding such a feature to the modern react interface.

Depending on the level of implementation you are looking for, you could potentially contribute this feature, or perhaps fund one of the developers to assign priority to it?

**Reply by jesseyu1984918** [reply]
May I know how much it is needed to fund this feature?

**Reply by jesseyu1984918** [reply]
Can anyone point me to the BOM html and it's backend funciton, I may be able to contribute to this but would for sure buy the developer a beer if he can enable this feature.
